server/api.py

```diff
@@ -14,6 +15,8 @@
     allow_methods=["*"],  # Allows all methods
     allow_headers=["*"],  # Allows all headers
 )
+args = parse_args()
+db_path = args.sqlite_file_path
```

Comment:
lets just use `args.sqlite_file_path` in the below lines instead of creating a variable

Reply:
done, thanks